addons/google_calendar/models/google_calendar.py

```diff
@@ -253,58 +249,60 @@ def generate_data(self, cr, uid, event, isCreating=False, context=None):
         if not event.active:
             data["state"] = "cancelled"
 
-        if not self.get_need_synchro_attendee(cr, uid, context=context):
+        if not self.get_need_synchro_attendee():
             data.pop("attendees")
         return data
 
-    def create_an_event(self, cr, uid, event, context=None):
-        gs_pool = self.pool['google.service']
-        data = self.generate_data(cr, uid, event, isCreating=True, context=context)
+    def create_an_event(self, event):
+        GsPool = self.env['google.service']
+        data = self.generate_data(event, isCreating=True)
 
-        url = "/calendar/v3/calendars/%s/events?fields=%s&access_token=%s" % ('primary', urllib2.quote('id,updated'), self.get_token(cr, uid, context))
+        url = "/calendar/v3/calendars/%s/events?fields=%s&access_token=%s" % ('primary', urllib2.quote('id,updated'), self.get_token())
         headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
         data_json = simplejson.dumps(data)
 
-        return gs_pool._do_request(cr, uid, url, data_json, headers, type='POST', context=context)
+        return GsPool._do_request(url, data_json, headers, type='POST')
 
-    def delete_an_event(self, cr, uid, event_id, context=None):
-        gs_pool = self.pool['google.service']
+    def delete_an_event(self, event_id):
```

Comment:
generic comment: decorator is missing from most of the methods where it should have been defined, otherwise it will throw traceback if they are called using old api/external script